### PR TITLE
feat: add Help Icon to "Setup Wallet Connection"

### PR DESCRIPTION
**Pull request by kennym**
# Description

Show an info button that triggers a toast with instructions on how to connect to Alby Hub.

## Context

Inspired because of #104 

## Screenshots

image

### CODE REVIEW

File: pages/settings/wallets/WalletConnection.tsx [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
 
 export function WalletConnection() {
   const hasConnection = useAppStore((store) => !!store.nwcClient);
   const walletIdWithConnection = useAppStore((store) =>
     store.wallets.findIndex((wallet) => wallet.nostrWalletConnectUrl),
   );
-  const [isScanning, setScanning] = React.useState(false);
-  const [isConnecting, setConnecting] = React.useState(false);
+  const [isScanning, setIsScanning] = React.useState(false);
+  const [isConnecting, setIsConnecting] = React.useState(false);
```
Comment (kennym) [inline pages/settings/wallets/WalletConnection.tsx:26]: this fixes react warning `useState call is not destructured into value + setter pair`

File: pages/settings/wallets/WalletConnection.tsx [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
 import Loading from "~/components/Loading";
 import QRCodeScanner from "~/components/QRCodeScanner";
 import Screen from "~/components/Screen";
+import { Button } from "~/components/ui/button";
+import { useBalance } from "~/hooks/useBalance";
+import { useInfo } from "~/hooks/useInfo";
+import { errorToast } from "~/lib/errorToast";
+
+import { nwc } from "@getalby/sdk";
+import { Nip47Capability } from "@getalby/sdk/dist/NWCClient";
```
Comment (kennym) [inline pages/settings/wallets/WalletConnection.tsx:18]: Sorry about these import order changes - I have `prettier` enabled by default.

Maybe we should add a `.prettierrc` in a separate PR?

File: components/ToastConfig.tsx [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
 
 export const toastConfig: ToastConfig = {
+  info: (props: BaseToastProps) => (
+    <BaseToast
+      {...props}
+      text2NumberOfLines={2}
+      text1Style={{ fontSize: 16 }}
+      text2Style={{ fontSize: 14 }}
+    />
+  ),
```
Comment (kennym) [inline components/ToastConfig.tsx:22]: I found the default text size for the info toast was unreadable, so I made it a little bit bigger.

File: pages/settings/wallets/WalletConnection.tsx (hunk 2 above)
Comment (kennym) [reply pages/settings/wallets/WalletConnection.tsx:18]: There already is one, but it's pretty empty

**Comment by reneaaron** [comment]
Thanks for the PR! Closing in favor like we discussed in #110 🙌

File: pages/settings/wallets/WalletConnection.tsx (hunk 2 above)
Comment (rolznz) [reply pages/settings/wallets/WalletConnection.tsx:18]: It shouldn't matter if it's pretty empty - it should use prettier sane defaults. It was added recently though, and we didn't run a full format. We should do that. I will create a separate issue.

File: pages/settings/wallets/WalletConnection.tsx (hunk 1 above)
Comment (rolznz) [reply pages/settings/wallets/WalletConnection.tsx:26]: I haven't seen this warning, weird. Do you know what plugin this is coming from?

**Pull request closed without merging**